src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+const elements = [...document.querySelectorAll('.population')];
+
+const values = elements.map((element) => {
+  const number = parseInt(element.textContent.replace(/,/g, ''), 10);
+
+  if (isNaN(number)) {
+    return;
```

Comment:
Returning `undefined` when `number` is `NaN` can lead to issues in the `reduce` function. Consider returning `0` or filtering out `undefined` values before the reduction.